examples/todomvc/src/element.rs

```diff
@@ -0,0 +1,235 @@
+use crate::dbg;
+extern crate wasm_bindgen;
+use wasm_bindgen::prelude::*;
+use wasm_bindgen::JsCast;
+
+/// Wrapper for `web_sys::Element` to simplify calling different interfaces
+pub struct Element {
```

Comment:
Should this `#[derive(Clone)]` for more ergonomic usage in Rust? (it's sort of like an `Rc` basically)

Reply:
I added this, not sure if it can be utilised in the current code though.